samples/dashboards/edit-mode/dragdrop-disabled/demo.js

```diff
@@ -0,0 +1,43 @@
+Dashboards.board('container', {
+    editMode: {
+        enabled: true,
+        dragDrop: {
+            enabled: false
+        },
+        contextMenu: {
+            enabled: true
```

Comment:
The context menu is enabled by default, so the flag `enabled` is not necessary here. (The problem is in others demo, so please fix in everywhere).

I'd prefer to setup as simple demo as possible.

Reply:
Thanks! [On my end it is disabled by default](https://jsfiddle.net/3mq9haut/1/).  The same for `editMode.enabled`. Could you confirm what it is supposed to be by default?

Reply:
You are right, sorry!